tests/test_flows.py

```diff
@@ -375,3 +384,41 @@ async def test_run_success_missing_run_name(
             ],
         )
         assert result == {"prefect-task": {"cell": "output"}}
+
+
+class TestJobsRunsIdSubmitAndWaitForCompletion:
```

Comment:
Can you also add tests for the three exceptions that can be raised from this flow?

Reply:
Yes, definitely need to get those added. Admittedly, I'm not very experienced at mocking API requests. Could I get some help with this step?

Reply:
I added tests for skipped, terminated, and time out